Extractor/Config.cs

```diff
@@ -642,6 +642,23 @@ public class SubscriptionConfig
         /// Log bad subscription datapoints
         /// </summary>
         public bool LogBadValues { get; set; } = true;
+
+        /// <summary>
+        /// Server namespace nodes the should be subscribed to a rebrowse upon changes to their values.
+        /// </summary>
+        public ServerNamespacesToRebrowseConfig? ServerNamespacesToRebrowse { get; set; }
+    }
+    public class ServerNamespacesToRebrowseConfig
+    {
+        /// <summary>
+        /// Indicate if the extractor should turn on subscription to namespaces.
+        /// </summary>
+        public bool Subscribe { get; set; } = false;
```

Comment:
That would also mean changing this to be slightly more specific, maybe `namespace-publication-date` or something.

Reply:
This would indicate that we only allow rebrowse triggers on specific nodes?

Reply:
Yeah, I figured we can add systems to subscribe to more nodes later. So have

`namespace-publication-date` subscribe to all namespace publication date nodes, unless filtered by `namespaces`, then we could add, in the future, a list of additional NodeIds that the user can subscribe to.

Reply:
Got it.

Reply:
If each node id is unique and we know the nodes we're targeting (e.g. with `namespace-publication-date`), I don't think we still need the `namespaces` filter. What do you think?